FAQ.md

```diff
@@ -52,12 +54,11 @@ ansible-playbook play.yml --tags flash
 
 ## What is required to bringup a location?
 
-1. Create a location folder at `/group_vars/` and fill in at least `general.yml`, `networks.yml` and `owm.yml`.
-2. Create a folder for every OpenWrt device at the location under `/host_vars/`. Paste the `base.yml` in there.
-3. Run the image creation as shown in the commands above (image will be in `./tmp/images/`).
-4. Flash the image to your router.
-5. Secure the router by setting a root password using SSH or the web interface.
-6. Done!
+1. Create a location file at `/locations/`. You might want to copy an existing location to make your start more easy.
```

Comment:
should be something like in the `locations` directory but not `/locations/` as that would refer to a totally different location in the file system.